core/src/main/java/org/apache/iceberg/view/ViewMetadata.java

```diff
@@ -85,6 +85,22 @@ default Integer currentSchemaId() {
   @Nullable
   String metadataFileLocation();
 
+  default String property(String property, String defaultValue) {
```

Comment:
On Similar convention of [TableMetadata](https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/TableMetadata.java#L481-L495) .  This is not related to the Hive-View changes. If it's extra change we can safely remove that.

Reply:
I don't think we really need these methods